core/templates/domain/exploration/WrittenTranslationObjectFactory.ts

```diff
@@ -20,34 +20,51 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
+export const WRITTEN_TRANSLATION_TYPE_HTML = 'html';
+export const WRITTEN_TRANSLATION_TYPE_UNICODE = 'unicode';
+
+type WrittenTranslationType = typeof WRITTEN_TRANSLATION_TYPE_UNICODE |
+  typeof WRITTEN_TRANSLATION_TYPE_HTML;
+
 export interface ITranslationBackendDict {
-  'html': string;
+  'data_format': WrittenTranslationType;
```

Comment:
I'm not sure about the terminology here. Why don't you call it a "data format" instead of changing the name to "type", here and elsewhere (in all the constants, field names, etc.)?

Better to have one name for the same concept throughout the codebase.

Reply:
Changed translationType => dataFormat